transform/shopify_dbt/models/source.yml

```diff
@@ -0,0 +1,16 @@
+# The source file lets the user define the data currently stored in dwh.
+# "- name" is the what dbt uses to refrence the source in the models.
+# "schema" is interchangeable with the BigQuery concept dataset.
+# "database" is interchangeable with the BigQuery concept of project.
+#  https://docs.getdbt.com/docs/build/sources
+
+version: 2
+
+sources:
+  - name: shopify
+    database: dlthub-analytics
```

Comment:
dlt will inject same database the pipeline is using. you may remove that

Reply:
I will remove the database entry.